css/_welcome_page_settings_toolbar.scss

```diff
@@ -0,0 +1 @@
+/** Insert custom CSS for any additional content in the welcome page settings toolbar **/
```

Comment:
@muscat1 In order for this file to be included in the output css shouldn't we import it in main.scss?

Reply:
Hmm... Actually I don't think this is used in our branding too. Is this correct?

Reply:
We are not currently using it in branding since we are reusing some classes from _welcome_page_content.scss, but I will add it to main.scss in order to decouple the styles meant for the additional components.